ui/app/AppLayouts/Communities/panels/SharedAddressesPanel.qml

```diff
@@ -83,6 +84,14 @@ Control {
             return false
         }
 
+        property var tokenCountMap: new Map()
+        property int tokenCountMapRevision: 0
```

Comment:
There is no need to use this additional `tokenCountMapRevision` property here. `tokenCountMap` is created only once and re-assigned, so the change signal is emitted. Expression may depend directly on `d.tokenCountMap`